packages/build-utils/src/schemas.ts

```diff
@@ -31,6 +31,11 @@ export const functionsSchema = {
           type: 'string',
           maxLength: 256,
         },
+        cron: {
+          type: 'string',
+          minLength: 9,
```

Comment:
Does this represent "* * * * *"? Is that the minimum possible cron string?

Reply:
Exactly, 5 fields + 4 spaces is the minimum